tonic/src/transport/channel/endpoint.rs

```diff
@@ -325,12 +326,30 @@ impl Endpoint {
         )
     }
 
+    /// Set the local address
+    ///
+    /// This sets  the IP address the client will use. By default we let hyper select the IP address.
+    /// ```
+    /// # use std::net::IpAddr;
+    /// # use std::str::FromStr;
+    /// # use tonic::transport::Endpoint;
+    /// # let mut builder = Endpoint::from_static("https://example.com");
+    /// # builder.local_address(IpAddr::from_str("127.0.0.1").expect("Unable to parse IP address"));
+    /// ```
```

Comment:
As the contents of this example are all hidden, they will not be displayed. And this example itself is redundant.

```suggestion
```